.github/workflows/php-cs-fixer.yml

```
@@ -0,0 +1,27 @@
+
+name: php-cs-fixer
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+
```

Remove excessive spaces in these workflow YAML files, the indentation(jobs section) is incorrect in this file.
```suggestion
name: php-cs-fixer

on:
  pull_request:
  workflow_dispatch:
```

---

FIxed